languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -34344,5 +34397,241 @@ Andere Projekte</example>
+                <message>Möchten Sie ein Komma hinter der Infinitivgruppe einfügen? (In den meisten Fällen ist dies optional)</message>
+                <suggestion>\1,</suggestion>
+                <example correction="auszudenken," type="incorrect">Sich einen Alternativplan <marker>auszudenken</marker> ist häufig schwierig.</example>
+            </rule>
+        </rulegroup>
+    </category>
+    <category id="EMPFOHLENE_RECHTSCHREIBUNG" name="Empfohlene Rechtschreibung laut Duden">
+        <rulegroup id="F_ANSTATT_PH" name="Worte mit F anstatt Ph">
+            <url>http://www.duden.de/hilfe/rechtschreibung</url>
+            <rule>
+                <pattern>
+                    <marker>
+                    <token regexp="yes" case_sensitive="yes"> Phantas.* </token>
+                    </marker>
+                </pattern>
+                <message>Möchten Sie die vom Duden empfohlene Schreibweise <suggestion><match no="1" regexp_match="Phantas(.*)" regexp_replace="Fantas$1"/></suggestion> verwenden?</message>
+                <example correction="Fantasie" type="incorrect">Er hat eine blühende <marker>Phantasie</marker>.</example>
+		<example correction="Fantasien" type="incorrect">Er entwickelte wilde <marker>Phantasien</marker>.</example>
+            </rule>
+            <rule>
+                <pattern>
+                    <marker>
+                    <token regexp="yes" case_sensitive="yes"> phantas.* </token>
+                    </marker>
+                </pattern>
+                <message>Möchten Sie die vom Duden empfohlene Schreibweise <suggestion><match no="1" regexp_match="phantas(.*)" regexp_replace="fantas$1"/></suggestion> verwenden?</message>
+                <example correction="fantastische" type="incorrect">Es ist eine <marker>phantastische</marker> Landschaft.</example>
+		<example correction="fantasievolle" type="incorrect">Er lieferte eine <marker>phantasievolle</marker> Beschreibung.</example>
+            </rule>
+            <rule>
+                <pattern>
+                    <marker>
+                    <token regexp="yes"> geograph.* </token>
+                    </marker>
+                </pattern>
+                <message>Möchten Sie die vom Duden empfohlene Schreibweise <suggestion><match no="1" regexp_match="(.)eograph(.*)" regexp_replace="$1eograf$2"/></suggestion> verwenden?</message>
+                <example correction="geografischen" type="incorrect">Er vertrieb sich die Zeit mit <marker>geographischen</marker> Studien.</example>
+                <example correction="Geografie" type="incorrect">Sie hat <marker>Geographie</marker> studiert.</example>
+            </rule>
+        </rulegroup>
+        <rulegroup id="EMPFOHLENE_GETRENNTSCHREIBUNG" name="Empfohlene Getrenntschreibung">
+            <url>http://www.duden.de/hilfe/rechtschreibung</url>
+            <rule>
+                <pattern>
+                    <marker>
+                    <token> zuhause </token>
```

Comment:
Please remove the whitespace around the ` zuhause ` (similar for other rules), just so we have it coherent.